scripts/scripts.js

```diff
@@ -1406,8 +1390,8 @@ async function loadLazy() {
   loadBlocks(main);
 
   if (getLanguage() === 'kr' || getLanguage() === 'jp') {
-    await loadTaxonomy();
-    lcpUpdateForTaxonomy();
+    const taxElements = document.querySelectorAll('.article-category a, .featured-article-card-category a');
+    await loadTaxonomy(taxElements);
   }
```

Comment:
This is from previous PR that I merged, I thought we would need to handle LCP separately but more I think through this, this is just more simple and making sense for now.